docs/operator-manual/reconcile.md

```diff
@@ -111,3 +111,59 @@ data:
     # actually changing in content.
     - .status.conditions[].lastTransitionTime
 ```
+
+## Tracking Dependent Resources
+
+Dependent resources by default are not being tracked. Therefore, we cannot generate any hash of those objects and utilize 
+the `ignoreResourceUpdates` configuration.
+
+If you want to track the dependent object and apply the `ignoreResourceUpdates` configuration, you can add 
+`argocd.argoproj.io/apply-resources-update=true` annotation in the dependent resources manifest:
```

Comment:
I think the concepts of tracked resources and watched resources here are mixed up. Also the user facing documentation is leaking the implementation. 

```suggestion

## Ignoring updates for untracked resources

ArgoCD will only apply `ignoreResourceUpdates` configuration to tracked resources of an application. This means dependant resources, such as a `ReplicaSet` and `Pod` created by a `Deployment`, will not ignore any updates and trigger a reconcile of the application for any changes.

If you want to apply the `ignoreResourceUpdates` configuration to an untracked resource, you can add the
`argocd.argoproj.io/ignore-resource-updates=true` annotation in the dependent resources manifest.
```